mapsforgesrv/src/main/java/com/telemaxx/mapsforgesrv/MapsforgeSrv.java

```diff
@@ -88,27 +89,18 @@ public MapsforgeSrv(String[] args) throws Exception {
 		}
 		connector.setAcceptQueueSize(mapsforgeConfig.SERVERACCEPTQUEUESIZE);
 		connector.setIdleTimeout(mapsforgeConfig.getIdleTimeout());
-		if (mapsforgeConfig.getListeningInterface().toLowerCase().equals("all")) { //$NON-NLS-1$
-			connector.setPort(mapsforgeConfig.getPortNumber());
-		} else if (mapsforgeConfig.getListeningInterface().toLowerCase().equals("localhost")) { //$NON-NLS-1$
-			connector.setPort(mapsforgeConfig.getPortNumber());
+		connector.setPort(mapsforgeConfig.getPortNumber());
+		if (mapsforgeConfig.getListeningInterface().toLowerCase().equals("localhost"))  //$NON-NLS-1$
 			connector.setHost("127.0.0.1");
-		} else {
-			logger.error("unkown Interface, only \"all\" or \"localhost\" , not " //$NON-NLS-1$
-					+ mapsforgeConfig.getListeningInterface());
-			System.exit(1);
-		}
 		server.addConnector(connector);
-
-		MapsforgeHandler mapsforgeHandler = new MapsforgeHandler(mapsforgeConfig, pool, queue);
-		server.setHandler(mapsforgeHandler);
 		try {
+			logger.info("################ STARTING SERVER ################");
 			server.start();
 		} catch (BindException e) {
 			logger.error("Stopping server", e); //$NON-NLS-1$
 			System.exit(1);
 		}
-		logger.info("> server listening on '"+mapsforgeConfig.getListeningInterface().toLowerCase()+":" + mapsforgeConfig.getPortNumber()+"'"); //$NON-NLS-1$
+		logger.debug("> server listening on '"+mapsforgeConfig.getListeningInterface().toLowerCase()+":" + mapsforgeConfig.getPortNumber()+"'"); //$NON-NLS-1$

```

Comment:
shouldn't we better use logger.info?
when i see this line, i know the server is up and waiting.

Reply:
`logger.info("> server listening on ...` is redundant with builtin jetty info log `org.eclipse.jetty.server.AbstractConnector:360@main - Started ServerConnector@5ffead27{HTTP/1.1, (http/1.1, h2c)}{127.0.0.1:8484}` this is why I downgrade it to debug

Reply:
the message from jetty is so technical, nerdy ;-)

i like more : "server is up and listening on 'all:8080'"